### DISCUSSION TITLE
Feature request: Can you implement Global Events HR40127/8

### DISCUSSION THREAD

**Discussion by sbeukers** [WillCodeForCats/solaredge-modbus-multi#357] [Ideas]
Hi,

First of all thanks for this great integration!

I have a feature request:
Can you implement Holding Register 40127 / 40128 Evt 2 bitfield32 Global Events

![image](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/65311410/5f63de54-71ec-497f-9197-1b0b363cb35e)

![image](https://github.com/WillCodeForCats/solaredge-modbus-multi/assets/65311410/0ff94a0a-33bf-4008-94c8-6c59a2757831)

Thank you so much in advance!

Sjoerd

**Comment by WillCodeForCats** [comment]
They are already implemented, but you have to have a SolarEdge Synergy inverter.

On all other inverters these registers are part of the Meter 1 block (specifically the manufacturer name string), not global events.

Global events cannot be implemented on non-Synergy inverters.

If your inverter is a Synergy type it will appear as sensor "MMPPT Events". If your inverter does not support it then the entity is not created.

**Reply by sbeukers** [reply]
Ah tanks! 

It's clear to me now!
